pallets/subtensor/src/lib.rs

```diff
@@ -57,6 +57,7 @@ pub mod subnet_info;
 extern crate alloc;
 pub mod migration;
 
+#[deny(missing_docs)]
```

Comment:
We should just do a crate-wide deny by using `#![deny(missing_docs)]`

Reply:
it will check all functions and structs. we can create new issues and PRs for these. make easier for review.